Spelling Fix (#36)
ballerina-platform/ballerina-dev-website: README.md

# About Ballerina.io

> Ballerina is an open source programming language and platform for cloud-era application programmers to easily write software that just works.

Reviewer: ```suggestion
Ballerina is an open source programming language and a platform for cloud-era application programmers to easily write software.
```

I feel "that just works" part is redundant.
